static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -410,6 +411,20 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
     }
   }
 
+  /**
+   * Register listeners so that any active pins are deselected when a user clicks
+   * or focuses on the search bar
+   */
+  setupSearchBarListeners () {
+    const searchBarInput = document.querySelector('#yxt-SearchBar-input--SearchBar');
```

Comment:
Can we use `this._container` instead? I think we want the Orchestrator scoped to elements inside its specified container

Reply:
Do we want this selector to be a configuration option? It seems this selector could change if a different `name` is given in the search bar options

Reply:
Ya, i'll try this._container. Also, I think there's another CSS class I can target instead which won't be susceptible to name changes